hw/drivers/crypto/src/mbedtls_aes_alt.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <os/mynewt.h>
+
+#if MYNEWT_VAL(MBEDTLS_AES_ALT)
+
+#include "crypto/crypto.h"
+#include "crypto/aes_alt.h"
+
+void
+mbedtls_aes_init(mbedtls_aes_context *ctx)
+{
+    memset(ctx, 0, sizeof(*ctx));
+    ctx->crypto = (struct crypto_dev *) os_dev_open("crypto", OS_TIMEOUT_NEVER,
+            NULL);
+}
```

Comment:
I probably would assert() here if ctx->crypto is NULL. User might've forgotten to create the crypto device. Although the system would crash when the first crypto operation is done. So not absolutely necessary.